File: ios/Classes/Configuration.swift
import Foundation

final class Configuration {
    public static let shared = Configuration()
    
    var channelName: String
    
    private init(apiToken: String = "", groupIds: [String] = [], lensId: String = "", channelName: String = "camerakit_flutter", isHideCloseButton: Bool = false) {
        self.channelName = channelName
    }
}

File: android/src/main/kotlin/com/camerakit/camerakit_flutter/Configuration.kt
package com.camerakit.camerakit_flutter

//
//  Configuration.kt
//  com.camerakit.camerakit_flutter
//
//  Copyright © 2023 DevCrew I/O
//
class Configuration private constructor(
    var channelName: String = "camerakit_flutter",
    var isHideCloseButton: Boolean = false
) {

    companion object {
        private var instance: Configuration? = null

        fun getInstance(): Configuration {
            if (instance == null) {
                instance = Configuration()
            }

            return instance!!
        }
    }

}

